readme.md

```diff
@@ -26,29 +30,31 @@ This package contains scripts that help build client code:
 
 ```js
 // src/main.js
-console.log('Welcome');
+console.log("Welcome");
 ```
 
-`$ inline-scripts src/index.html out/index.html`
+`$ inline-script-tags src/index.html out/index.html`
 
 ```html
 <!-- out/index.html -->
 <p>Welcome</p>
-<script>console.log('Welcome');</script>
+<script>
```

Comment:
Although this is easier to read, it's actually inaccurate. The script doesn't generate the new lines and indentation.